apps/dashboard/src/components/project-navigation/index.js

```diff
@@ -21,24 +21,29 @@ const Tab = {
 };
 
 const ProjectNavigation = ( { activeTab, disableTitleEditor, projectId } ) => {
-	const { saveAndUpdateProject } = useDispatch( STORE_NAME );
+	const { setEditorTitle } = useDispatch( STORE_NAME );
 
-	const project = useSelect(
-		( select ) => select( STORE_NAME ).getProject( projectId ),
+	const projectTitle = useSelect(
+		( select ) => select( STORE_NAME ).getEditorTitle(),
 		[ projectId ]
 	);
 
-	const updateTitle = ( title ) =>
-		saveAndUpdateProject( projectId, {
-			name: title,
-			title,
-		} );
+	const { setEditorContentChanged } = useDispatch( STORE_NAME );
```

Comment:
Let's concatenate both of these `useDispatch` into a single call - or delete this if you decide to go with my suggestion below where `setEditorContentChanged` would become unnecessary here.